### PR TITLE
Optimised finding first managed resource.

### PR DESCRIPTION
**Pull request by sbusk**

### CODE REVIEW

File: whois-commons/src/main/java/net/ripe/db/whois/common/dao/jdbc/JdbcManagedAttributeDao.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    public boolean hasManagedResource(final CIString orgId) {
+        final int orgObjectId = jdbcTemplate.queryForObject(
+            "select object_id from organisation where organisation = ?",
+                new Object[] { orgId },
+                Integer.class
+        );
+
+        return executeStreaming(
+                jdbcTemplate,
+                "select l.object " +
```
Comment (eshryane) [inline whois-commons/src/main/java/net/ripe/db/whois/common/dao/jdbc/JdbcManagedAttributeDao.java:43]: Uppercase SQL keywords for readibility (and consistency)

File: whois-api/src/main/java/net/ripe/db/whois/api/rest/search/ManagedAttributeSearch.java [hunk 2]
```diff
@@ -17,24 +16,25 @@
...
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.dao.DataAccessException;
 import org.springframework.stereotype.Component;
 
 import javax.annotation.Nullable;
-import java.util.Optional;
-import java.util.stream.Stream;
+
+import static net.ripe.db.whois.common.rpsl.AttributeType.ORG;
```
Comment (eshryane) [inline whois-api/src/main/java/net/ripe/db/whois/api/rest/search/ManagedAttributeSearch.java:23]: Inconsistent imports (some fields are imported, some are not)

File: whois-commons/src/main/java/net/ripe/db/whois/common/dao/jdbc/JdbcManagedAttributeDao.java (hunk 1 above)
Comment (sbusk) [reply whois-commons/src/main/java/net/ripe/db/whois/common/dao/jdbc/JdbcManagedAttributeDao.java:43]: Ok.

File: whois-api/src/main/java/net/ripe/db/whois/api/rest/search/ManagedAttributeSearch.java (hunk 2 above)
Comment (sbusk) [reply whois-api/src/main/java/net/ripe/db/whois/api/rest/search/ManagedAttributeSearch.java]: Done.

**Pull request merged**